docs/sources/about.md

```diff
@@ -14,15 +14,14 @@ weight: 100
 
 # Introduction to Grafana Agent
 
-Grafana Agent is a vendor-neutral, batteries-included telemetry collector. It
-is designed to be flexible, performant, and compatible with multiple ecosystems
-such as Prometheus and OpenTelemetry.
+Grafana Agent is a flexible, high performance, vendor-neutral telemetry collector. It's fully compatible with the most popular
+open source observability standards such as OpenTelemetry (OTel) and Prometheus.
```

Comment:
nit:

[Semantic line breaks](https://sembr.org/) are better for future line based diffing in the GitHub UI and result in the same HTML output.

```suggestion
Grafana Agent is a flexible, high performance, vendor-neutral telemetry collector. It's fully compatible with the most popular open source observability standards such as OpenTelemetry (OTel) and Prometheus.
```